L29-opencensus-filter.md

```diff
@@ -24,7 +24,12 @@ We propose that the OpenCensus filter (C++ grpc filter) which currently resides
 
 The major changes introduced by this are as follows:
   1) A new optional dependency on OpenCensus for bazel builds. This will be a new build target in the Bazel build file. Currently only the Bazel build will be supported.  There will be no git submodule for OpenCensus, and no make/cmake support (this may be added at a later time).
-  2) A new API call will be introduced to enable the OpenCensus filter for tracing and stats collection. This must be called to register the plugin mechanism and initialize OpenCensus. RegisterOpenCensusPlugin() will reside in src/cpp/ext/filters/census/grpc_plugin.h.  There will also be an optional API call to register default views for OpenCensus.
+  2) Two new public API calls will be introduced to enable the OpenCensus filter for tracing and stats collection and register default views. The registration call must be made to register the plugin mechanism and initialize OpenCensus. The call to register default views is optional and does not need to be used if the user will be registering their own views.
+
+  * The public API calls will reside in include/grpcpp/opencensus.h :
+
+        void RegisterOpenCensusPlugin();
+        void RegisterGrpcViewsForExport();
```

Comment:
This should be renamed to something like `RegisterOpenCensusViewsForExport()`, so that it's clear to gRPC users that this function is related to OpenCensus.

Reply:
Done.